### PR TITLE
feat(web-app): first pass content migration

### LINKED ISSUE #751: Populate all existing non-asset pages
Based on https://github.com/carbon-design-system/carbon-platform/issues/439

- Update left nav
- Create and populate all existing non-asset pages (using remote MDX where possible)
- Stub out empty mdx pages

_Fix broken image https://next.carbondesignsystem.com/assets/guidelines/themes/overview_

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #751

#### Changelog

Migrate content based on https://github.com/carbon-design-system/carbon-platform/issues/439
Add zenhub icon to icons package (added here but forgot to include image https://github.com/carbon-design-system/carbon-platform/pull/824/files#diff-ceffaba23a1444cf2b14884575d69841ffee93aef36ef1ab0fde9f4e35caaea5R44)

### Pages not included in this PR

- https://github.com/carbon-design-system/carbon-platform/issues/779 ~Developing~ 
- https://github.com/carbon-design-system/carbon-platform/issues/754 Designing > Get started 
- https://github.com/carbon-design-system/carbon-platform/issues/780 Contribute > Documentation

### CODE REVIEW

**Comment by aubrey-oneal** [comment]
Here is my review:

(New page) How Carbon Works
✅ Text content (yes, in the future we'll update once we know our naming conventions better)
✅ Image
✅ Anchor links
✍️ Update link to components catalog  /**assets**/components
✍️ Add a link to the GH schema doc under 'Creating Consistency': The Carbon Design System now follows a schema (link)

Platform roadmap
✅ Text content (in the future we might change the intro text but it is good here)
✅ Anchor links: none (in the future we might add these)
✅ Images
✅ Links to GH and Airtable

Releases
✅ Text content (in the future we might change the intro text but it is good here)
✅ Anchor links: none (in the future we might add these)
✍️ Images: missing Zenhub icon on Roadmap card
✍️  Link: Should 'V11 release page' link to http://localhost:3000/migrating/guide/overview or somewhere else?
🟨 Not related to this PR, but the banners on old CDS release sites don't point towards the lates version; they only point to the next version.

Articles
✅ Everything looks good (minus the MediumPost component in progress)

Case studies
✍️  Text content: There is variation in the image caption font size and missing bolded words that appear in CDS.
✅ Anchor links
✅ Video plays and media looks good
✍️ Bug: An issue is open to fix tree nav collapsing when user clicks on a page (i.e. under case studies.)
✍️ Icon: missing Cloud icon for Resource card on consistency-in-the-cloud
✍️ Icon: missing Bee icon for Resource card on ibm-commerce-platform

Meetups
✅ Text content
✅ Links
✅ Took out Carbon insider

Get started
✅ Page shell created

Figma
✅ New and old body content good
✍️ Update page header to Figma
✍️ Update Component docs link to: https://help.figma.com/hc/en-us/articles/360038662654-Guide-to-components-in-Figma
✍️ Change 'Join Figma Friday's' to 'Connect'
✍️ Add anchor link to Connect


Other design kit pages
✅ Content is good
✍️ Update page header to name of tool 

Developing
✍️ No need to nest LHN as there is only one page: 'Developing'

Contributing
✅ Content is good

Elements


Libraries
FAQ
Contact

File: services/web-app/pages/about-carbon/how-carbon-works.mdx [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+lenses**. Our goal is to help makers find assets that comply with platform requirements, are
+convenient to implement, and are consistent with design patterns across the company.
+
+For the first time, makers will be able to find and re-use assets across all IBM teams, regardless
+of who maintains them.
+
+Additionally, makers can quickly find assets that are compliant with their own product or team
+through local lenses.
+
+Head over to the [component catalog](/components) to see the many assets shared between product and
```
Comment (aubrey-oneal) [inline services/web-app/pages/about-carbon/how-carbon-works.mdx:49]: Update link to components catalog /assets/components

File: services/web-app/pages/about-carbon/how-carbon-works.mdx [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+digital teams. Progress on the new platform can be found in our
+[roadmap](/about-carbon/platform-roadmap).
+
+## Creating consistency
+
+The Carbon Design System now follows a schema to document each resource in a standardized way. This
```
Comment (aubrey-oneal) [inline services/web-app/pages/about-carbon/how-carbon-works.mdx:55]: Add a link to the GH schema doc under 'Creating Consistency': The Carbon Design System now follows a schema (link)

File: services/web-app/pages/about-carbon/releases.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { NextSeo } from 'next-seo'
+
+import RemoteMdxLoader from '@/components/remote-mdx-loader'
+import { getRemoteMdxData } from '@/lib/github'
+
+const RemoteMdxPage = ({ source }) => {
```
Comment (aubrey-oneal) [inline services/web-app/pages/about-carbon/releases.js:12]: Link: Should 'V11 release page' link to http://localhost:3000/migrating/guide/overview or somewhere else?

File: services/web-app/pages/about-carbon/case-studies/ibm-commerce-platform.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { NextSeo } from 'next-seo'
+
+import RemoteMdxLoader from '@/components/remote-mdx-loader'
+import { getRemoteMdxData } from '@/lib/github'
+
+const RemoteMdxPage = ({ source }) => {
```
Comment (aubrey-oneal) [inline services/web-app/pages/about-carbon/case-studies/ibm-commerce-platform.js:12]: Text content: There is variation in the image caption font size and missing bolded words that appear in CDS.

Comment (aubrey-oneal) [inline services/web-app/pages/about-carbon/case-studies/ibm-commerce-platform.js:12]: Icon: missing Bee icon for Resource card on ibm-commerce-platform

File: services/web-app/pages/about-carbon/case-studies/consistency-in-the-cloud.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { NextSeo } from 'next-seo'
+
+import RemoteMdxLoader from '@/components/remote-mdx-loader'
+import { getRemoteMdxData } from '@/lib/github'
```
Comment (aubrey-oneal) [inline services/web-app/pages/about-carbon/case-studies/consistency-in-the-cloud.js:10]: Icon: missing Cloud icon for Resource card on consistency-in-the-cloud

File: services/web-app/pages/designing/figma.mdx [hunk 6]
```diff
@@ -0,0 +1,176 @@
+---
+title: Design kits
```
Comment (aubrey-oneal) [inline services/web-app/pages/designing/figma.mdx:2]: Update page header to Figma

File: services/web-app/pages/designing/axure.mdx [hunk 7]
```diff
@@ -0,0 +1,147 @@
+---
+title: Design kits
```
Comment (aubrey-oneal) [inline services/web-app/pages/designing/axure.mdx:2]: Update header to Axure

File: services/web-app/pages/designing/adobe-xd.mdx [hunk 8]
```diff
@@ -0,0 +1,221 @@
+---
+title: Design kits
```
Comment (aubrey-oneal) [inline services/web-app/pages/designing/adobe-xd.mdx:2]: Update header to Adobe XD

File: services/web-app/pages/designing/figma.mdx [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+
+Included in the library are all 32 of the Carbon components and their variants. To insert a
+component, go to the **Asset** panel and find the component you would like to use. Drag it from the
+asset panel onto the canvas.
+
+View the name of the component in the right sidebar. If the component has variants, you'll see
+fields underneath the component name to configure the properties and values of that component set.
+
+For more help on how to use Figma components, see the
+[Figma docs](https://help.figma.com/hc/en-us/articles/360056440594-Create-and-use-variants#Use_variants).
```
Comment (aubrey-oneal) [inline services/web-app/pages/designing/figma.mdx:144]: Update Component docs link to: https://help.figma.com/hc/en-us/articles/360038662654-Guide-to-components-in-Figma

File: services/web-app/pages/developing/index.mdx [hunk 10]
```diff
@@ -0,0 +1,6 @@
+---
+title: Developing
```
Comment (aubrey-oneal) [inline services/web-app/pages/developing/index.mdx:2]: No need to nest LHN as there is only one page: 'Developing'

File: services/web-app/pages/elements/accessibility/overview.js [hunk 11]
```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { NextSeo } from 'next-seo'
```
Comment (aubrey-oneal) [inline services/web-app/pages/elements/accessibility/overview.js:7]: Bee icon missing in Resource card

**Review by aubrey-oneal — CHANGES_REQUESTED** [review]
@alisonjoseph , I left in-line comments as well as a compiled list. Since some of my comments are about remote mdx pages, I couldn't put them in a specific line.

File: services/web-app/pages/about-carbon/releases.js (hunk 3 above)
Comment (alisonjoseph) [reply services/web-app/pages/about-carbon/releases.js:12]: Good catch, this is where its linking to now. https://carbondesignsystem.com/migrating/guide/overview/
where will this content be on the new site?

**Review by aubrey-oneal — APPROVED** [review]

**Review by andreancardona — APPROVED** [review]
Looks really good! did a full test and didn't find any bugs / visual regressions 💯

File: services/web-app/pages/about-carbon/case-studies/consistency-in-the-cloud.js [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+export const getStaticProps = async () => {
+  const mdxSource = await getRemoteMdxData(
+    {
+      host: 'github.com',
+      org: 'carbon-design-system',
+      repo: 'carbon-website',
+      library: 'carbon-website',
+      ref: 'main'
+    },
+    '/src/pages/case-studies/consistency-in-the-cloud.mdx'
```
Comment (francinelucca) [inline services/web-app/pages/about-carbon/case-studies/consistency-in-the-cloud.js:34]: this one icon doesnt load for me, are we good with that? 
image

File: services/web-app/pages/about-carbon/case-studies/ibm-commerce-platform.js [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+export const getStaticProps = async () => {
+  const mdxSource = await getRemoteMdxData(
+    {
+      host: 'github.com',
+      org: 'carbon-design-system',
+      repo: 'carbon-website',
+      library: 'carbon-website',
+      ref: 'main'
+    },
+    '/src/pages/case-studies/ibm-commerce-platform.mdx'
```
Comment (francinelucca) [inline services/web-app/pages/about-carbon/case-studies/ibm-commerce-platform.js:34]: icon doesn't load 
image

File: services/web-app/pages/faq/index.mdx [hunk 14]
```diff
@@ -0,0 +1,168 @@
...
+Carbon has a core team of designers and front-end developers dedicated to developing and supporting
+the system. There are also countless other designers and developers who contribute back to
+[carbon-components](https://github.com/carbon-design-system/carbon-components#contributors) and
+[carbon-react](https://github.com/carbon-design-system/carbon#contributors).
+
+## Contributing
+
+### How can I contribute and/or propose new components or ideas?
+
+Check out our [contribution overview](/contributing/overview) to learn all about the different ways
```
Comment (francinelucca) [inline services/web-app/pages/faq/index.mdx:37]: this one takes me to 404

File: services/web-app/pages/faq/index.mdx [hunk 15]
```diff
@@ -0,0 +1,168 @@
...
+to contribute. Thanks for the help!
+
+### I see a bug. How do I report it?
+
+First, make sure the problem is reproducible. Once confirmed, open an issue in the appropriate
+GitHub repo from the following list. We will address the bug as soon as we can. If you have a fix
+for the bug, please feel free to submit a PR for it.
+
+For larger changes, please see our [contribution overview](/contributing/overview).
```
Comment (francinelucca) [inline services/web-app/pages/faq/index.mdx:46]: this one takes me to 404

File: services/web-app/pages/faq/index.mdx [hunk 16]
```diff
@@ -0,0 +1,168 @@
...
+
+- Microsoft Edge latest
+- Firefox latest
+- Chrome latest
+- Safari latest
+
+### What languages are the components written in?
+
+Carbon is built React first. We also support vanilla JS, Angular, Vue, and Svelte implementations.
+Check out [Frameworks](/developing/frameworks/react) to learn more about what you need to get
```
Comment (francinelucca) [inline services/web-app/pages/faq/index.mdx:103]: this one takes me to 404

File: services/web-app/pages/faq/index.mdx [hunk 17]
```diff
@@ -0,0 +1,168 @@
...
+### Where do I find mobile components?
+
+We are currently working on a Carbon mobile Sketch kit. This kit is a work in progress but is an
+opportunity for you to preview what we're working on and provide early feedback. Find the kit on the
+[Work in progress](/whats-happening/work-in-progress) page.
+
+### Where do I find the Experimental components and patterns?
+
+The patterns and components that were in Experimental are now in the
+[Community assets](/community/overview) section. Community assets include work developed and
```
Comment (francinelucca) [inline services/web-app/pages/faq/index.mdx:115]: this one takes me to 404

File: services/web-app/pages/faq/index.mdx [hunk 18]
```diff
@@ -0,0 +1,168 @@
...
+
+Carbon is built React first. We also support vanilla JS, Angular, Vue, and Svelte implementations.
+Check out [Frameworks](/developing/frameworks/react) to learn more about what you need to get
+started with Carbon implementations.
+
+### Where do I find mobile components?
+
+We are currently working on a Carbon mobile Sketch kit. This kit is a work in progress but is an
+opportunity for you to preview what we're working on and provide early feedback. Find the kit on the
+[Work in progress](/whats-happening/work-in-progress) page.
```
Comment (francinelucca) [inline services/web-app/pages/faq/index.mdx:110]: this one takes me to 404

File: services/web-app/pages/faq/index.mdx [hunk 19]
```diff
@@ -0,0 +1,168 @@
...
+maintained by the individuals and teams who use Carbon. You can help elevate these resources by
+contributing feedback, code, and design.
+
+Note that some of the content in Community assets is hosted on internal IBM sites and is only
+accessible to IBMers.
+
+### Where do I find charts and data visualization?
+
+Carbon charts and their corresponding assets and documentation are now in the
+[Data visualization](/data-visualization/getting-started) section.
```
Comment (francinelucca) [inline services/web-app/pages/faq/index.mdx:125]: this one takes me to 404

File: services/web-app/pages/contact/index.mdx [hunk 20]
```diff
@@ -0,0 +1,97 @@
...
+
+### Carbon website
+
+As a first step, it's always good to explore the content on this website. The site is comprehensive
+and most of the guidelines and components are well documented.
+
+### Frequently asked questions
+
+Answers to the most common questions about the design system can be found in the
+[Carbon FAQs](faqs).
```
Comment (francinelucca) [inline services/web-app/pages/contact/index.mdx:33]:
```suggestion
[Carbon FAQs](faq).
```

File: services/web-app/pages/contribute/overview.js [hunk 21]
```diff
@@ -0,0 +1,44 @@
...
+export const getStaticProps = async () => {
+  const mdxSource = await getRemoteMdxData(
+    {
+      host: 'github.com',
+      org: 'carbon-design-system',
+      repo: 'carbon-website',
+      library: 'carbon-website',
+      ref: 'main'
+    },
+    '/src/pages/contributing/overview.mdx'
```
Comment (francinelucca) [inline services/web-app/pages/contribute/overview.js:34]: this link takes me to 404 (/help/contact-us)

File: services/web-app/pages/contribute/bugs-and-requests.js [hunk 22]
```diff
@@ -0,0 +1,44 @@
...
+export const getStaticProps = async () => {
+  const mdxSource = await getRemoteMdxData(
+    {
+      host: 'github.com',
+      org: 'carbon-design-system',
+      repo: 'carbon-website',
+      library: 'carbon-website',
+      ref: 'main'
+    },
+    '/src/pages/contributing/bugs-and-requests.mdx'
```
Comment (francinelucca) [inline services/web-app/pages/contribute/bugs-and-requests.js:34]: image
takes me to 404 (/help/contact-us)

File: services/web-app/pages/contribute/components.js [hunk 23]
```diff
@@ -0,0 +1,44 @@
...
+export const getStaticProps = async () => {
+  const mdxSource = await getRemoteMdxData(
+    {
+      host: 'github.com',
+      org: 'carbon-design-system',
+      repo: 'carbon-website',
+      library: 'carbon-website',
+      ref: 'main'
+    },
+    '/src/pages/contributing/component/index.mdx'
```
Comment (francinelucca) [inline services/web-app/pages/contribute/components.js:34]: the links to components here are pointing to /[asset]/usage which 404s in platform, we don't have that tab yet, are we cool with that? 🤔

**Review by francinelucca — COMMENTED** [review]
mostly lgtm, pointed out some icons that don't load for me and some 404s, figured I'd stop here and wait to hear back on the 404s before I go through the full thing 🤣

File: services/web-app/components/mdx/components.js [hunk 24]
```diff
@@ -99,6 +99,7 @@ const components = {
   DoDontRow,
   GifPlayer,
   Glossary,
+  GlossaryComponent: Glossary,
```
[the remark this answers is not in the document]
Comment (alisonjoseph) [reply services/web-app/components/mdx/components.js:102]: It's named GlossaryComponent in the Carbon website currently, and we're pulling that page remotely. I can just copy that page into an mdx page and rename locally so we don't have to do this.

File: services/web-app/pages/about-carbon/case-studies/consistency-in-the-cloud.js (hunk 12 above)
Comment (alisonjoseph) [reply services/web-app/pages/about-carbon/case-studies/consistency-in-the-cloud.js:34]: Yea, remote svgs aren't loading, being tracked here https://github.com/carbon-design-system/carbon-platform/issues/767

File: services/web-app/pages/about-carbon/case-studies/ibm-commerce-platform.js (hunk 13 above)
Comment (alisonjoseph) [reply services/web-app/pages/about-carbon/case-studies/ibm-commerce-platform.js:34]: https://github.com/carbon-design-system/carbon-platform/issues/767

File: services/web-app/components/mdx/components.js (hunk 24 above)
[the remark this answers is not in the document]
Comment (francinelucca) [reply services/web-app/components/mdx/components.js:102]: ah ok, can we rename Glossary -> GlossaryComponent locally and just use one?

[the remark this answers is not in the document]
Comment (alisonjoseph) [reply services/web-app/components/mdx/components.js]: fixed. just copied this page over to a new mdx file, no need to pull remotely and make things more complicated.

File: services/web-app/pages/contribute/components.js (hunk 23 above)
Comment (alisonjoseph) [reply services/web-app/pages/contribute/components.js:34]: Checked in with Aubrey, we're good with 404s in remote pages for now 👍

**Review by francinelucca — APPROVED** [review]
💃

**Pull request merged**